src/engine/ruleset/integrations/windows/decoders/windows-sysmon.yml

```diff
@@ -117,41 +280,41 @@ normalize:
     - file.pe.product: $windows.EventData.Product
 
   # DNS event
- - check: $event.code=="22"
+ - check: $event.code == 22
    map:
     - network.protocol: dns
 
- - check: $windows.EventData.SignatureStatus==Valid
+ - check: $windows.EventData.SignatureStatus == Valid
    map:
     - file.code_signature.valid: True
 
- - check: $windows.EventData.Initiated==True
+ - check: $windows.EventData.Initiated == True
```

Comment:
Are you sure that $windows.EventData.Initiated is a boolean, I think it is a string, please check this. The checks are typed